docs/sensors.rst

```diff
@@ -18,7 +18,7 @@ The classes derive from :py:class:`Sensor` and provide helper functions
 specific to the corresponding sensor type. Each of the functions makes
 sure the sensor is in the required mode and then returns the specified value.
 
-.. ~autogen doc-special-sensor-classes
+..
```

Comment:
One of the reasons I liked autogen: I bet next time we change something in a special sensor, we forget to document it :).

Reply:
True...there is always a tradeoff :)